spec/acceptance/class_spec.rb

```diff
@@ -2,7 +2,11 @@
 
 require 'spec_helper_acceptance'
 
-describe 'puppetboard class' do
+RSpec.configure do |c|
+  c.filter_run_excluding "broken_#{fact('os.name')}#{fact('os.release.major')}_puppet#{ENV['BEAKER_PUPPET_COLLECTION'].sub(%r{puppet}, '')}".downcase.to_sym => true
```

Comment:
This `ENV['BEAKER_PUPPET_COLLECTION'].sub(%r{puppet}, '')` is awful, but `fact('puppetversion').split('.').first` seems not to be an option unfortunately.

Reply:
> `fact('puppetversion').split('.').first` seems not to be an option unfortunately.

I think `fact()` doesn't send `--puppet`. Perhaps it should be an option.

Reply:
No more relevant now in support file.